java/tv/sage/ModuleGroup.java

```diff
@@ -293,27 +293,32 @@ else if (cryptoKeyObj instanceof Object[])
         }
       }
       file.createNewFile();
-      sage.FastRandomFile widgetDBout = new sage.FasterRandomFile(file, cryptoKey == null ? "rwd" : "rwdc", sage.Sage.I18N_CHARSET);
-      if (cryptoKey != null) widgetDBout.setCrypto(cryptoKey);
+      sage.io.SageDataFile widgetDBout;
+
+      if (cryptoKey == null)
```

Comment:
This confused me a little:
`widgetDBout.writeUnencryptedByte(cryptoKey != null ? (byte) 0x36 : (byte)0x20); // < 0x2F to signify unencrypted`

The code clearly shows 0x20, but your comment shows 0x2F. I presume the code is correct since the comment is just a comment. Should I update the comment?